tox.ini

```diff
@@ -487,6 +491,8 @@ changedir =
     datastore_pysolr: tests/datastore_pysolr
     datastore_redis: tests/datastore_redis
     datastore_rediscluster: tests/datastore_rediscluster
+    datastore_valkey: tests/datastore_valkey
+    datastore_valkeycluster: tests/datastore_valkeycluster
```

Comment:
It looks like this was copied over from redis but there are no valkeycluster tests in your PR.

Reply:
@hmstepanek removed.